configuration/configuration_blackbox_test.go

```diff
@@ -31,7 +40,7 @@ func TestMain(m *testing.M) {
 
 func resetConfiguration() {
 	var err error
-	config, err = configuration.GetConfigurationData()
+	config, err = configuration.NewConfigurationData("../config.yaml")
```

Comment:
@sbose78 have in mind that the deployment via OC will use ENV variables to set the configuration, as done now.